commands/login.go

```diff
@@ -99,14 +99,8 @@ func runLogin(cmd *cobra.Command, args []string) error {
 }
 
 func validateLogin(gatewayURL string, user string, pass string) error {
-	tr := &http.Transport{
```

Comment:
Given our offline conversation I guess this would have stopped the proxy from being used for login?

Reply:
correct, yes I tested the current release and it doesn't proxy the login